scripts/flexible-server/common-functions.sh

```diff
@@ -1,23 +1,23 @@
 #!/bin/bash
 
-# Function that uses the subscription input to get set variables for later use and gather all flexible sql servers within the subscription for shutdown
-function get_subscription_flexible_sql_servers() {
-  SUBSCRIPTION_ID=$(jq -r '.id' <<< $subscription)
-  SUBSCRIPTION_NAME=$(jq -r '.name' <<< $subscription)
-  az account set -s $SUBSCRIPTION_ID
-  FLEXIBLE_SERVERS=$(az resource list --resource-type Microsoft.DBforPostgreSQL/flexibleServers --query "reverse(sort_by([?tags.autoShutdown == 'true'], &to_string(contains(id, 'replica'))))" -o json)
-}
-
 # Function that accepts the flexible sql server json as input and sets variables for later use to stop or start the flexible sql server
 function get_flexible_sql_server_details() {
   RESOURCE_GROUP=$(jq -r '.resourceGroup' <<< $flexibleserver)
+  log "$RESOURCE_GROUP"
```

Comment:
these needed? should probably say what is being logged before hand? e.g. resource group?